packages/cosmic-swingset/src/chain-main.js

```diff
@@ -30,7 +30,10 @@ import { makeShutdown } from '@agoric/internal/src/node/shutdown.js';
 
 import * as STORAGE_PATH from '@agoric/internal/src/chain-storage-paths.js';
 import * as ActionType from '@agoric/internal/src/action-types.js';
-import { BridgeId as BRIDGE_ID } from '@agoric/internal';
+import {
+  BridgeId as BRIDGE_ID,
+  BridgeKeys as BRIDGE_KEYS,
```

Comment:
we should be consistent with the casing. if it's supposed to be ALL_CAPS then let's change it in the `internal` package. But I don't think it should be. `BridgeId` is a namespace and the values are ALL_CAPS. 

i.e. `BridgeId.PROVISION_SMART_WALLET`, which is much easier to read than `BRIDGE_ID.PROVISION_SMART_WALLET`

so I request,
```suggestion
  BridgeId,
  BridgeKeys,
```

If this isn't agreeable please point to some documentation for the style in the PR currently

Reply:
It was cargo-culted from other code.